metrics/cli/test/services/firebase/adapter/firebase_cli_service_adapter_test.dart

```diff
@@ -393,16 +414,20 @@ void main() {
     );
 
     test(
-      ".version() shows the version information",
+      ".version() returns the version information",
```

Comment:
🏷️ 
```suggestion
      ".version() returns the process result of the cli version command",
```
What do you think?